src/schema/objects/metadata.yaml

```diff
@@ -3014,6 +3190,26 @@ SEEGChannelCount:
     Number of SEEG channels.
   type: integer
   minimum: 0
+SampleCoordinateSystem:
+  name: SampleCoordinateSystem
+  display_name: Coordinate System of the Gaze Position
+  description: |
+    Coordinate system of the gaze position recordings.
+    Generally eye-tracker are set to use `"gaze-on-screen"` coordinate system but you may use
+    `"eye-in-head"` or `"gaze-in-world"` or other alternatives of your choice.
+    If you use the standard `"gaze-on-screen"`, it is RECOMMENDED to use this
+    exact label.
+  type: string
+SampleCoordinateUnits:
```

Comment:
As far as I can tell, the `SampleCoordinateUnits` field should be a single `string`, indicating a single Unit for _all_ eyetrack channels, which is fine for X/Y eyegaze data but if you include pupil size data then this might not work (since pupil data are usually in a different unit than eyegaze data).

On that note: the `pupil_size` field description in `src/schema/objects/columns.yaml` says:

> Pupil size or area of the recorded eye, in the units specified in the corresponding metadata sidecar.

But to my point above, I see no way to specify the unit of pupil data here. So should we update `SampleCoordinteUnits`  to allow for an array of strings, or should we add another field like `PupilUnit`? Or am I missing something?

Reply:
Ahh.. I see an example now 🙂 :

```
# sub-01_task-visualSearch_recording-eye1_physio.json

"pupil_size": {
        "Description": "Pupil area of the recorded eye as calculated by the eye-tracker in arbitrary units (see EyeLink's documentation for conversion).",
        "Units": "a.u."
    },
```

Is a way to make it more clear that fields for each channel name can be added to this JSON file, folks like me? 🙂